src/python/espressomd/io/mpiio.py

```diff
@@ -0,0 +1,71 @@
+from ..script_interface import PScriptInterface
+
+class Mpiio(object):
+    """MPI-IO object.
+
+    Used to output particle data using MPI-IO to binary files.
+
+    .. note::
+        See the write and read methods for documentation.
+    """
+    def __init__(self):
+        self._instance = PScriptInterface("ScriptInterface::MPIIO::MPIIOScript")
+
+    def write(self, prefix=None, positions=False, velocities=False, types=False, bonds=False):
+        """MPI-IO write.
+
+        Outputs binary data using MPI-IO to files named "prefix.*" where * in:
+         - head: Information about fields that are dumped,
+         - pref: Information about processes: 1 int per process,
+         - id: Particle ids: 1 int per particle,
+         - pos: Position information (if dumped): 3 doubles per particle,
+         - vel: Velocity information (if dumped): 3 doubles per particle,
+         - typ: Type information (if dumped): 1 int per particle,
+         - bond: Bond information (if dumped): variable amount of data,
+         - boff: Bond offset information (if bonds are dumped): 1 int per particle.
+
+        .. note::
+            Do not read the files on a machine with a different architecture!
+
+        Parameters
+        ----------
+        prefix : :obj:`str`
+                   Common prefix for the filenames.
```

Comment:
just to be picky... please align the explanation with the type description above